packages/performance/src/services/api_service.ts

```diff
@@ -109,11 +113,30 @@ export class Api {
     );
   }
 
-  requiredApisAvailable(): boolean {
-    if (fetch && Promise && this.navigator && this.navigator.cookieEnabled) {
+  async requiredApisAvailable(): Promise<boolean> {
```

Comment:
The change looks good!
Can you please add a test for this function as it is not trivial anymore?